impedance_control_controller:
  type: advanced_robotics_control/ImpedanceControlController
  joint_names:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6

  root_link: world
  tip_link: end_effector_link